setup.py

```diff
@@ -18,5 +18,5 @@
     author_email='hello@dev.ngo',
     license='BSD',
     zip_safe=False,
-    install_requires=['django>=1.8']
+    install_requires=['django>=4.2']
```

Comment:
do we need to include `psycopg` in here?

Reply:
Technically - yes, though it hasn't been a problem as all the projects will have it already (for v2 that is) 😂 

However I've added it.